File: C#/Contoh Looping/Uncounted Loop/While.cs
using System;

class main
{
    static void Main(string[] args)
    {
        char  ulang = 'y';
        int i = 0;

        while (ulang == 'y')
        {
            Console.WriteLine();
            Console.WriteLine("Masukkan huruf y untuk contoh perulangan");
            Console.WriteLine();
            Console.Write("Masukkan huruf : ");
            ulang = char.Parse(Console.ReadLine());
            i++;
        }
        Console.WriteLine();
        Console.WriteLine("Selesai");

    }
}

File: Kotlin/Contoh Looping/Uncounted Loop/While.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var input = Scanner(System.`in`)
    var ulang = true
    var i = 0
    
    while (ulang) {
        println(" ")
        println("Masukkan huruf y untuk contoh perulangan")
        println(" ")
        print("Masukkan huruf : ")
        var jawab = input.next()

        if( jawab.equals("y") ){
            ulang = true
        }
        else{
            ulang = false
        }
        i++
    }
    println(" ")
    println("Selesai")

}
